### PR TITLE
docs: add CephObjectStoreUser CR example

### PR DESCRIPTION
**Pull request by jhoblitt**

### CODE REVIEW

**Review by galexrt — CHANGES_REQUESTED** [review]
Thanks for the docs PR! One small comment regarding a potential structure change :smile:

File: README.md [hunk 1]
```diff
@@ -20,6 +20,27 @@ Due to the closure of Koor Technologies, Inc. this repository has been made to c
     radosgw-admin user info --uid extended-ceph-exporter | jq '.keys[0].secret_key'
     ```
 
+If using Rook to manage rgw, the admin user may also be created using a `CephOjectStoreUser` resource:
```
[the remark this answers is not in the document]
Comment (jhoblitt) [reply README.md:23]: That makes sense to me.  I suspect that many users of this exporter are also using rook (as am I).

[the remark this answers is not in the document]
Comment (galexrt) [reply README.md:23]: @jhoblitt Do you want to add the "Rook" section in this PR or should I add it?

**Comment by galexrt** [comment]
I have added the Rook section to the README based on your text, thanks for providing the instructions!

**Comment by jhoblitt** [comment]
> I have added the Rook section to the README based on your text, thanks for providing the instructions!

Thank you. I apologize for having not gotten to this. Something had managed to come up every time I was about to look at it.

**Comment by galexrt** [comment]
No worries, your instructions will definitely help a lot of Rook users get "extended" metrics more easily, thanks! :smile:

**Pull request closed without merging**